tests/RobotFramework/config_management/conf_child_mgmt.robot

```diff
@@ -0,0 +1,186 @@
+${payl_exec}            '{"status": "executing", "path": "/home/pi/config1", "type": "config1", "reason": null}'
+${payl_succ}            '{"status": "successful", "path": "/home/pi/config1", "type": "config1", "reason": null}'
+
+
+*** Test Cases ***
+Prerequisite Parent
+    Parent Connection
+    Delete child related content                    #Delete any previous created child related configuration files/folders on the parent device
+    Set external MQTT bind address                  #Setting external MQTT bind address which child will use for communication 
+    Set external MQTT port                          #Setting external MQTT port which child will use for communication Default:1883
+    Reconnect c8y                                   #Disconnect and Connect to c8y
+    Restart Configuration plugin                    #Stop and Start c8y-configuration-plugin
+    Close Connection
+Prerequisite Child
+    Child device delete configuration files         #Delete any previous created child related configuration files/folders on the child device
+Prerequisite Cloud
+    GET Parent ID                                   #Get the Parent ID from the cloud
+    GET Parent name                                 #Get the Parent name from the cloud
+    GET Child ID                                    #Get the Child ID from the cloud
+    GET Child name                                  #Get the Child name from the cloud
+    # Validate child Name                             #This is to check the existence of the bug: https://github.com/thin-edge/thin-edge.io/issues/1569
+Child device bootstrapping
+    Startup child device                            #Setting up/Bootstraping of a child device
+Snapshot from device
+    Request snapshot from child device              #Using the cloud command Get snapshot from device
+    Child device response on snapshot request       #Child device is sending 'executing' and 'sussess' MQTT responses
+Child device config update
+    Send configuration to device                    #Using the cloud command Send configuration to device
+    Child device get configuration file             #Child device is sending 'executing' and 'sussess' MQTT responses
+
+
+*** Keywords ***
+Parent Connection
+    Open Connection     ${PARENT_IP}
+    Login               ${USERNAME}    ${PASSWORD}
+Child Connection
+    Open Connection     ${CHILD_IP}
+    Login               ${USERNAME}    ${PASSWORD}
+Set external MQTT bind address
+    ${rc}=    Execute Command    sudo tedge config set mqtt.external.bind_address ${PARENT_IP}    return_stdout=False    return_rc=True    #Set the URL of your Cumulocity IoT tenant
+    Should Be Equal    ${rc}    ${0}
+Set external MQTT port
+    ${rc}=    Execute Command    sudo tedge config set mqtt.external.port 1883    return_stdout=False    return_rc=True    #Set the URL of your Cumulocity IoT tenant
```

Comment:
The comment of this line "#Set the URL of your Cumulocity IoT tenant" is not applicable.